sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -387,6 +387,10 @@ public class SentryOptions {
 
   private @NotNull IMemoryCollector memoryCollector = NoOpMemoryCollector.getInstance();
 
+  // TODO this should default to false on the next major
```

Comment:
Do you agree with this approach of having it default to `true` until the next major then change it to the desired default of `false`?

Reply:
Sounds good 👍